src/sql/workbench/services/notebook/browser/models/cell.ts

```diff
@@ -718,6 +718,8 @@ export class CellModel extends Disposable implements ICellModel {
 				if (this._outputs.length > 0) {
 					for (let i = 0; i < this._outputs.length; i++) {
 						if (this._outputs[i].output_type === 'execute_result') {
+							// deletes transient node in the serialized JSON
```

Comment:
I'd probably include why we're deleting it - that's going to be a lot more useful of a message for people looking through the code =)